app/views/notes/_form.html.erb

```diff
@@ -0,0 +1,42 @@
+<div><h3><%= form_title %></h3></div>
+
+<div>
+  <% if note.errors.any? %>
+    <% note.errors.full_messages.each do |msg| %>
+      <p> <%= msg %></p>
+    <% end %>
+  <% end %>
+</div>
+
+<div>
+  <%= form_for(note) do |f| %>
+    <div>
+      <label for="name">Name: </label>
+      <%= f.text_area :name, placeholder: "meeting_notes_2016-08-04", class: 'form-field' %>
+    </div>
+
+    <div>
+      <label for="body">Body: </label>
+      <%= f.text_area :body, placeholder: "Today we started planning...", class: 'form-field' %>
+    </div>
+
+    <div>
+      <label for="file_type">File Extension: </label>
+      <%= f.text_area :file_type, placeholder: ".txt", class: 'form-field' %>
```

Comment:
This should be a drop down with the options we currently allow, we don't want bad data by allowing a user to type a file extension that does not exist.